scripts/asos/iem_scraper_example.py

```diff
@@ -134,3 +205,37 @@ def main():
 if __name__ == "__main__":
     download_alldata()
     # main()
+
+    
+'''
+Example use case of using the construct_urls and pull_metar functions. 
+
+def main():
+    """ Takes start and finish dates from command line, then passes it to retrieve the metars. """
+
+    # Takes in user input for the start and end dates.
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--start', nargs='+', type=int)
+    parser.add_argument('--finish', nargs='+', type=int)
+    args = parser.parse_args()
+    start = datetime.datetime(*args.start)
+    finish = datetime.datetime(*args.finish)
+
+    metar = []
+    # Can pull in from file, just needs to be List or Tuple.
+    station = ["CYOW", "CYYZ"]
+    
+    url = construct_urls(station, start, finish)
+
+    for i in url:
```

Comment:
`i` is usually an `int` and not a string url in this case.  Suggested change:
```
urls = construct_urls(station, start, finish)
for url in urls:
  metar += pull_metar(url)
```

This logic still eludes me as you would have a string with repeated headers.